packaging/.itch.toml

```diff
@@ -0,0 +1,17 @@
+[[prereqs]]
+name = "net-4.7.1"
```

Comment:
Our minimum requirement is actually 4.7.2 currently, but according to https://github.com/itchio/itch/issues/2458 neither of these actually exist. Won't this PR depend on that being fixed?

Reply:
.NET 4.7.1 is supported https://github.com/itchio/butler/issues/184 as an undocumented feature. Left a comment as a reminder.

Reply:
Testing log:

* Installed the itch app on a fresh Windows VM that does not have a new enough .NET framework (I used a windows 8.1 image, which only has 4.0)
* Installed experimental windows portable build
* Attempt to run RedAlert.exe through the itch app: crashes on launch with `System.MissingMethodException: Method not found: '!!0[] System.Array.Empty()'.`.

⚠️ The itch.toml for that build defines a prereq on `net-4.6.2`, but the itch app did not prompt to install it. ⚠️ 

* Download and install .NET framework 4.7.1 from https://dotnet.microsoft.com/download/dotnet-framework/net471
* Attempt to run RedAlert.exe through the itch app: runs as expected (because release-20200503 only requires .NET 4.6.1)
* Tag a [fresh devtest build](https://github.com/pchote/OpenRA/releases/tag/devtest-20200927) with goal of demonstrating that .NET 4.7.1 is not sufficient, and will crash with something similar to above.
* Download the winportable build and extract over the itch install location.
* Attempt to run RedAlert.exe through the itch app: pleasantly surprised to see a windows error dialog saying that .NET 4.7.1 was not sufficient, with a link to download 4.8.

So there are two problems here:
* The prereqs definition on the live itch build isn't working, meaning that players with sufficiently old versions of the .NET framework will simply crash on startup.
* Declaring a prereq on 4.7.1, assuming it worked, would have itch automatically(?) install it but then still require the player to manually install a more recent version.

I'm not convinced this is a better user experience than having itch run the installer, which will bypass the time and bandwidth of installing the unnecessary 4.7.1 framework assuming that the prereqs issue was a fixable oversight, and the crashes if not.

Reply:
I suggest we simply make this depend on #17989, which will finally solve the .NET version nightmare by shipping the runtime files alongside the game.

Reply:
I asked nicely and we got .NET 4.7.2 support.